index.md

```diff
@@ -126,56 +121,61 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
+| 3 | howto/monitor/collect-metrics | [Example: Collect metrics](https://discourse.ubuntu.com/t/monitoring-grafana/17787)|
+| 3 | howto/monitor/monitor-status | [Example: Monitor status](https://discourse.ubuntu.com/t/monitoring-nagios/17788)|
 | 0 | | |
 | 1 | ref/landing | [Reference](https://discourse.ubuntu.com/t/reference/28828) |
-| 2 | ref/roadmap | [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359)|
-| 2 | ref/release-notes | [Release notes](https://discourse.ubuntu.com/t/release-notes/17842)|
+| 2 | ref/releases-versions | [Releases and versions](tbd) |
+| 3 | ref/roadmap | [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359)|
+| 3 | ref/release-notes | [Release notes](https://discourse.ubuntu.com/t/release-notes/17842)|
+| 3 | ref/supported-versions | [Supported versions](https://discourse.ubuntu.com/t/supported-versions/21046)|
+| 3 | ref/component-versions | [Component versions](https://discourse.ubuntu.com/t/component-versions/21413)|
 | 2 | ref/requirements | [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) |
-| 2 | ref/supported-versions | [Supported versions](https://discourse.ubuntu.com/t/supported-versions/21046)|
-| 2 | ref/component-versions | [Component versions](https://discourse.ubuntu.com/t/component-versions/21413)|
 | 2 | ref/provided-images | [Provided images](https://discourse.ubuntu.com/t/provided-images/24185)|
+| 2 | ref/license-information | [License information](https://discourse.ubuntu.com/t/license-information/36649) |
+| 2 | ref/supported-rendering-resources | [Supported rendering resources](https://discourse.ubuntu.com/t/supported-rendering-resources/37322) |
+| 2 | ref/supported-video-codecs | [Supported video codecs](https://discourse.ubuntu.com/t/supported-video-codecs/37323)|
 | 2 | ref/android-features | [Supported Android features](https://discourse.ubuntu.com/t/supported-android-features/28825)|
-| 2 | ref/sdks | [Anbox Cloud SDKs](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844)|
-| 2 | ref/api-reference | [API reference](https://discourse.ubuntu.com/t/api-reference/24339) |
+| 2 | ref/instance-types | [Instance types](https://discourse.ubuntu.com/t/instance-types-reference/17764)|
+| 2 | ref/ams-configuration | [AMS configuration](https://discourse.ubuntu.com/t/ams-configuration/20872)|
+| 2 | ref/application-manifest | [Application manifest](https://discourse.ubuntu.com/t/application-manifest/24197)|
+| 2 | ref/api-reference | [APIs](https://discourse.ubuntu.com/t/api-reference/24339) |
 | 3 | ref/ams-http-api | [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/)|
 | 3 | ref/anbox-https-api | [Anbox HTTP API](https://discourse.ubuntu.com/t/anbox-http-api-reference/17819)|
 | 3 | ref/anbox-stream-gateway | [Stream Gateway API](https://anbox-cloud.github.io/latest/anbox-stream-gateway/)|
 | 3 | ref/anbox-platform-sdk-api | [Anbox Platform SDK API](https://anbox-cloud.github.io/latest/anbox-platform-sdk/)|
```

Comment:
Shouldn't these be under `ref/api-reference`?

Reply:
The folder organisation and the navigation order is not 100% mapped 1:1. The order in navigation might need to be grouped together in order to provide a better user experience/understanding. We could redo the folder structure to be exactly the same but that would break URLs and unnecessarily introduce a lot of redirects. 

I still could do this but just wondering if it is worth the effort.

Reply:
Makes sense! I think it's fine if the folder organization is not 100% mapped if it's too much effort to get everything into place.